internal/test/e2e/cluster.go

```diff
@@ -560,7 +560,7 @@ func newTestAutopilot(masterKey utils.MasterKey, cfg config.Autopilot, bus *bus.
 
 	c := contractor.New(bus, bus, bus, bus, bus, cfg.RevisionSubmissionBuffer, cfg.RevisionBroadcastInterval, cfg.AllowRedundantHostIPs, l)
 	p := pruner.New(bus, l)
-	w := walletmaintainer.New(a, bus, l)
+	w := walletmaintainer.New(a, bus, 5, 5, l)
```

Comment:
I hated this myself but the only alternative was to update the config and add a `wallet` section, which we definitely don't want to do.

Reply:
Why not add functional options like we do in other places and set 10 and 100 as the default? Then `cmd/renterd/node.go` doesn't change at all and here we pass `WithNumOutputs(min, desired)`

Reply:
I considered it but it is inconsistent with how we instantiate the other dependencies of the autopilot and I don't like it when the logger is not the last argument (if it's an argument at all and not an option). I understand why you ask for it, but now we have to keep the logger out of the option because it would defeat the purpose and we'd have to update node.go anyway.